### ISSUE TITLE
Update project setup and ussage instructions

### ISSUE THREAD

**Issue by CodyBontecou** [TribeWorks/TribeWorks-hugo#2]
Following this build instructions is failing on my local machine.

**Specs:**
- OSX Version 10.13.6
- Hugo version v0.62.0

I'm unsure of what instruction #2 is referencing and this may be what I am doing wrong, but a git clone of this repo and then running `Hugo Server` results in this error

```
Error: module "hugo-material-banner" not found; either add it as a Hugo Module or store it in "/Users/codybontecou/MindViewLLC/TribeWorksSoftware/TribeWorks-hugo/themes".: module does not exist
```

**Comment by pavelbinar** [comment]
Yes, I think the instructions should be updated to be more in line with my original instructions:
https://github.com/BruceEckel/WinterTechForum-hugo/blob/master/README.md

`build.sh` is made for CI usage.

here is the proposed update:

## Requirements

- Git
- [Hugo](https://gohugo.io/getting-started/installing/) `v0.59.0`

## Setup

```bash
git clone git@github.com:TribeWorks/TribeWorks-hugo.git
cd TribeWorks-hugo
git clone https://github.com/MindviewLLC/hugo-material-banner.git themes/hugo-material-banner
```

## Development

```bash
hugo server
```

## Build

```bash
hugo server
```

The build output will be in the `public` folder.
